srcpkgs/libsoup/template

```diff
@@ -1,27 +1,32 @@
 # Template file for 'libsoup'
 pkgname=libsoup
-version=2.66.4
+version=2.68.3
 revision=1
 build_style=meson
 build_helper="gir"
-configure_args="-Dgssapi=false -Dintrospection=$(vopt_if gir true false)
- -Dvapi=$(vopt_if vala true false)"
+configure_args="-Dgssapi=disabled -Dintrospection=$(vopt_if gir enabled disabled)
+ -Dvapi=$(vopt_if vala enabled disabled)"
 hostmakedepends="gettext-devel glib-devel pkg-config $(vopt_if vala 'vala')"
-makedepends="libglib-devel libxml2-devel sqlite-devel libpsl-devel
- $(vopt_if vala 'vala-devel')"
+makedepends="libglib-devel libxml2-devel sqlite-devel libpsl-devel brotli-devel
+ glib-networking smbclient $(vopt_if vala 'vala-devel')"
 depends="glib-networking"
 short_desc="HTTP library implementation in C"
 maintainer="Enno Boland <gottox@voidlinux.org>"
 license="LGPL-2.1-or-later"
 homepage="https://wiki.gnome.org/Projects/libsoup"
-changelog="https://raw.githubusercontent.com/GNOME/libsoup/gnome-3-28/NEWS"
+changelog="https://gitlab.gnome.org/GNOME/libsoup/raw/master/NEWS"
 distfiles="${GNOME_SITE}/${pkgname}/${version%.*}/${pkgname}-${version}.tar.xz"
-checksum=8308984f1eee1c4f8c113a9c1763b2b22d981bd811b0cc82a9f3f1aa63228779
+checksum=534bb08e35b0ff3702f3adfde87d3441e27c12f9f5ec351f056fe04cba02bafb
 
 # Package build options
 build_options="gir vala"
 build_options_default="gir vala"
 
+if [ "$CROSS_BUILD" ]; then
+	configure_args+=" -Dtls_check=false"
+	hostmakedepends+=" smbclient"
```

Comment:
remove smbclient from makedepends, set it in hostmakedepends instead,

then remove this line.